### PR TITLE
T-1: Validate what time is selected "Today" on Time Logging page

### PR DESCRIPTION
**Pull request by nk-kaia**

### CODE REVIEW

File: cypress/integration/login.js [hunk 1]
```diff
@@ -20,5 +20,12 @@ describe('Login functionality', function() {
         cy.get('.tile.form').should('be.visible')
         cy.get('.user-info__title').contains('TestCon User 6')
         cy.get('.main-nav').find('li').should('have.length', 2)
+
+        let d = new Date();
+        let current_day = d.getDate();
+
+        cy.get('.calendar--selected').should('contain', current_day)
```
Comment (nikolaj-tolkaciov) [inline cypress/integration/login.js:27]: Nice, almost there! '.calendar--selected' is not the best selector for today's date, as this one will change the date value when you select another day in the calendar

**Review by nikolaj-tolkaciov — CHANGES_REQUESTED** [review]

File: cypress/integration/login.js (hunk 1 above)
Comment (nk-kaia) [reply cypress/integration/login.js:27]: Oh yeah a very good point! Didn't check that!

File: cypress/integration/login.js [hunk 2]
```diff
@@ -20,5 +20,12 @@ describe('Login functionality', function() {
         cy.get('.tile.form').should('be.visible')
         cy.get('.user-info__title').contains('TestCon User 6')
         cy.get('.main-nav').find('li').should('have.length', 2)
+
+        let d = new Date();
+        let current_day = d.getDate();
+
+        cy.get('.calendar--today').should('contain', current_day)
+
```
Comment (nikolaj-tolkaciov) [inline cypress/integration/login.js:28]: please keep the style consistent, if you want to leave an empty line after a test case, make the same changes for previous test cases as well.

**Review by nikolaj-tolkaciov — APPROVED** [review]
Nice! Almost done, only minor style consistency issues please fix it and you can merge this pull request

File: cypress/integration/login.js (hunk 2 above)
Comment (nk-kaia) [reply cypress/integration/login.js]: Thanks for your review, not sure if I got what you have said exactly but I tried to update my code the way as I understood it. ^^

File: cypress/integration/time-logging.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
+describe('Time-logging functionality', function() {
+
+    it('Should display the current day properly', function () {
+
+        const d = new Date();
+        const current_day = d.getDate();
+
+        cy.visit('/')
```
Comment (MonkeviciuteGerda) [inline cypress/integration/time-logging.js:8]: These lines are from another test, they are not required in here.

**Review by MonkeviciuteGerda — CHANGES_REQUESTED** [review]

**Pull request merged**